### PR TITLE
add --minconfig flags for cloudcore

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: zhangjie <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

 /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref: #1171 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
add --minconfig flags for cloudcore
```

### CODE REVIEW

**Comment by kadisi** [comment]
--minconfig output:

```
# cloudcore --minconfig

apiVersion: cloudcore.config.kubeedge.io/v1alpha1
kind: CloudCore
kubeAPIConfig:
  kubeConfig: /root/.kube/config
  master: ""
modules:
  cloudhub:
    nodeLimit: 10
    tlsCAFile: /etc/kubeedge/ca/rootCA.crt
    tlsCertFile: /etc/kubeedge/certs/edge.crt
    tlsPrivateKeyFile: /etc/kubeedge/certs/edge.key
    unixsocket:
      address: unix:///var/lib/kubeedge/kubeedge.sock
      enable: true
    websocket:
      address: 0.0.0.0
      enable: true
      port: 10000


# With --minconfig , you can easily used this configurations as reference.
# It's useful to users who are new to KubeEdge, and you can modify/create your own configs accordingly.
# This configuration is suitable for beginners.

```

**Comment by kevin-wangzefeng** [comment]
make cloud_lint failed

File: cloud/cmd/cloudcore/app/flags/flags.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package flags defines utility functions to handle command line flags
+// related to outputting config as referance.
```
Comment (kevin-wangzefeng) [inline cloud/cmd/cloudcore/app/flags/flags.go]: If this file is copied from Kubernetes, please clarify what is changed.
Otherwise please fix the file head

Comment (kadisi) [reply cloud/cmd/cloudcore/app/flags/flags.go]: done

**Pull request merged**